resources/views/masjid_profile/edit.blade.php

```diff
@@ -12,6 +12,16 @@
                     {!! FormField::text('masjid_name', ['required' => true, 'value' => old('masjid_name', Setting::get('masjid_name')), 'label' => __('masjid_profile.name')]) !!}
                     {!! FormField::textarea('masjid_address', ['required' => true, 'value' => old('masjid_address', Setting::get('masjid_address')), 'label' => __('masjid_profile.address')]) !!}
                     {!! FormField::text('masjid_google_maps_link', ['value' => old('masjid_google_maps_link', Setting::get('masjid_google_maps_link')), 'label' => __('masjid_profile.google_maps_link')]) !!}
+                    <div class="form-group">
+                        <label class="form-label" for="masjid_profile_image">Logo Masjid</label>
+                        <div class="custom-file">
+                          <input type="file" class="custom-file-input image" id="masjid_profile_image" name="masjid_profile_image">
+                          <label class="custom-file-label" for="masjid_profile_image">Choose file</label>
+                        </div>
+                        @if (Setting::get('masjid_image'))
+                            <img id="masjid_profile_image_show" class="img-fluid mt-2" src="{{ asset(Setting::get('masjid_image'))}}" alt="{{ Setting::get('masjid_name') ?? 'buku masjid'}}">
```

Comment:
1. `asset()` diganti dengan Storage::url(Setting::get('masjid_logo_path))
2. Di setting itu kita bisa tetapkan default valuenya jika value tidak ditemukan `Setting::get('key', 'default value')`.

```suggestion
                            <img id="masjid_profile_image_show" class="img-fluid mt-2" src="{{ Storage::url(Setting::get('masjid_image')) }}" alt="{{ Setting::get('masjid_name', config('masjid.name')) }}">
```